src/finfield.c

```diff
@@ -542,10 +543,10 @@ FF              FiniteField (
       else
```

Comment:
I am somewhat a fan of `{}` for the ifs (in particular for the one with the `else` on it). I realise that this code was here before, I just noticed it)